### ISSUE TITLE
capitalization from MS spelling disappears with Volpiano

### ISSUE THREAD

**Issue by annamorphism** [DDMAL/CantusDB#1258]
Proofreading a chant, I noticed that the melody-with-text sometimes seems to get rid of capital letters (?). 
I noticed it on the word "cuius":
<img width="635" alt="image" src="https://github.com/DDMAL/CantusDB/assets/67451875/c21d8181-134b-4bdf-804e-a40d9c959c93">

Are there other places where some sort of syllabification exception does this?

**Comment by jacobdgm** [comment]
@dchiller, this won't be a problem once we implement the new syllabification/alignment library, correct?

**Comment by dchiller** [comment]
> @dchiller, this won't be a problem once we implement the new syllabification/alignment library, correct?

Correct! Capitalization is maintained in syllabification in the new process.
